test/unit/slurm/test_slurm_config_generator.py

```diff
@@ -102,14 +102,17 @@ def test_generate_slurm_config_files_memory_scheduling(
 
 
 @pytest.mark.parametrize(
-    "slurm_accounting",
-    [(False), (True)],
+    "slurm_accounting, dbname_passed",
+    [(False, False), (True, False), (True, True)],
 )
-def test_generate_slurm_config_files_slurm_accounting(mocker, test_datadir, tmpdir, slurm_accounting):
+def test_generate_slurm_config_files_slurm_accounting(mocker, test_datadir, tmpdir, slurm_accounting, dbname_passed):
+    input_base_name = "sample_input"
```

Comment:
Instead of building the names programmatically (which requires inspecting various parts of this function), can we pass the files names via Parametrize
`("input_file, output_file", [...])`.

In this case for example, we had to introduce a new if condition in order to test a new parameter.

Reply:
Addressed.